rpc/tests/rpc.rs

```diff
@@ -224,15 +224,14 @@ mod tests {
     async fn test_block_transaction_count_by_hash_is_ok() {
         let kakarot_rpc = setup_kakarot_eth_rpc().await;
         let hash =
-            H256::from_str("0x0197be2810df6b5eedd5d9e468b200d0b845b642b81a44755e19047f08cc8c6e")
+            H256::from_str("0x0449aa33ad836b65b10fa60082de99e24ac876ee2fd93e723a99190a530af0a9")
```

Comment:
why are we changing the hash btw? what does it bring?

Reply:
Because now I am using the "get_block_with_txs" method and in the wiremock_utils file it has that [hash](https://github.com/sayajin-labs/kakarot-rpc/blob/abfb3d5564b930782a153fe4bfdb770dc59cd8a6/core/src/utils/wiremock_utils.rs#L131)